Sources/CubeStyleGuide/Model/ComponentSwatch.swift

```diff
@@ -8,29 +8,33 @@
 
 import SwiftUI
 
-public struct ComponentSwatch {
+public struct ComponentSwatch: Identifiable {
 
+    public var id: UUID
```

Comment:
Am teeeeempted to say supporting backwards compatibility would be good where `name` is the identifier. But don't feel too strongly about it